src/neuroconv/datainterfaces/behavior/medpc/medpc_helpers.py

```diff
@@ -0,0 +1,173 @@
+import numpy as np
+
+from neuroconv.utils import FilePathType
+
+
+def get_medpc_variables(file_path: FilePathType, variable_names: list) -> dict:
```

Comment:
IMO should mark these (and parent file) as private to indicate we are simply using them in our own interface code, not to be advertised or exposed for public use since often the return structures (such as the `dict` here) are specific to our own needs

Reply:
It makes sense for `_get_session_lines` to be private since it really only supposed to be used by `read_medpc_file`.  But I used `get_medpc_variables` directly in the Lerner Lab conversion to obtain a list of all the sessions I wanted to convert, so that should be public.  Similarly, I think `read_medpc_file` should be public in case any future conversions want to just read the medpc data and do some operation before writing to nwb.